docs/src/SUMMARY.md

```
@@ -0,0 +1,90 @@
+# Summary
+
+[Homepage](README.md)
+
+# User manual
```

markdownlint-cli2 (0.17.2)

Multiple top-level headings in the same document
null

(MD025, single-title, single-h1)